bot/admin/server/src/main/kotlin/verticle/ScenarioVerticle.kt

```diff
@@ -0,0 +1,330 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ai.tock.bot.admin.verticle
+
+import ai.tock.bot.HandlerNamespace
+import ai.tock.bot.admin.BotAdminService
+import ai.tock.bot.admin.mapper.ScenarioExceptionManager
+import ai.tock.bot.admin.mapper.ScenarioMapper.toScenarioActionHandlerResponse
+import ai.tock.bot.admin.mapper.ScenarioMapper.toScenarioGroup
+import ai.tock.bot.admin.mapper.ScenarioMapper.toScenarioGroupResponse
+import ai.tock.bot.admin.mapper.ScenarioMapper.toScenarioVersion
+import ai.tock.bot.admin.mapper.ScenarioMapper.toScenarioVersionResponse
+import ai.tock.bot.admin.model.scenario.*
+import ai.tock.bot.admin.service.ScenarioService
+import ai.tock.bot.handler.ActionHandlersRepository
+import ai.tock.shared.security.TockUser
+import ai.tock.shared.security.TockUserRole.*
+import ai.tock.shared.vertx.WebVerticle
+import io.vertx.ext.web.RoutingContext
+import mu.KLogger
+import mu.KotlinLogging
+
+/**
+ * ScenarioVerticle contains all the routes and actions associated with the scenarios
+ */
+open class ScenarioVerticle {
+
+    private val logger: KLogger = KotlinLogging.logger {}
+    //private val ScenarioServiceImpl: ScenarioServiceImpl by injector.instance()
+
+    private val botId = "botId"
+    private val groupId = "groupId"
+    private val versionId = "versionId"
+    private val tickStoryId = "tickStoryId"
+
+    private val importOneScenarioGroupPath    = "/bot/:$botId/scenarios/import/groups"
+    private val importManyScenarioVersionPath = "/bot/:$botId/scenarios/import/groups/:$groupId/versions"
+    private val createOneScenarioGroupPath    = "/bot/:$botId/scenarios/groups"
+    private val createOneScenarioVersionPath  = "/bot/:$botId/scenarios/groups/:$groupId/versions"
+
+    private val getAllScenarioGroupPath       = "/bot/:$botId/scenarios/groups"
+    private val getOneScenarioGroupPath       = "/bot/:$botId/scenarios/groups/:$groupId"
+    private val getAllScenarioVersionPath     = "/bot/:$botId/scenarios/groups/:$groupId/versions"
+    private val getOneScenarioVersionPath     = "/bot/:$botId/scenarios/groups/:$groupId/versions/:$versionId"
+
+    private val updateOneScenarioGroupPath    = "/bot/:$botId/scenarios/groups/:$groupId"
+    private val updateOneScenarioVersionPath  = "/bot/:$botId/scenarios/groups/:$groupId/versions/:$versionId"
+
+    private val deleteOneScenarioGroupPath    = "/bot/:$botId/scenarios/groups/:$groupId"
+    private val deleteOneScenarioVersionPath  = "/bot/:$botId/scenarios/groups/:$groupId/versions/:$versionId"
+
+    private val getAllActionHandlerPath     = "/bot/:$botId/dialog-manager/action-handlers"
+
+    /**
+     * Declaration of routes and their appropriate handlers
+     */
+    fun configureScenario(webVerticle: WebVerticle) {
+        logger.info { "Configure Scenario Verticle" }
```

Comment:
```suggestion
        logger.info { "Configure ScenarioVerticle" }
```
Dans les autres logs les verticle name sont attachés il me semble